sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/StateDictionaryJsonConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using SystemJson = System.Text.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
+{
+    internal class StateDictionaryJsonConverter : JsonConverter<IReadOnlyDictionary<string, object>>
```

Comment:
What if customers have a `IReadOnlyDictionary<string, object>` property as part of their state?